buildscripts/ci/backend/docker/setup.sh

```diff
@@ -82,7 +83,30 @@ apt-get install -y --no-install-recommends \
   "${apt_packages_runtime[@]}" \
   "${apt_packages_ffmpeg[@]}"
 
-
 # DISTROS
 HERE="$(cd "$(dirname "$0")" && pwd)"
 bash $HERE/install_mu.sh
+
+# Google Fonts installation
+TEMP_DIR=$(mktemp -d)
+FONTS_DIR="$HOME/.local/share/fonts"
+
+echo "Downloading Google Fonts..."
+ZIP_URL="https://github.com/google/fonts/archive/main.zip"
+mkdir -p "$TEMP_DIR"
+cd "$TEMP_DIR"
+curl -L -o fonts.zip "$ZIP_URL"
+
+echo "Unpacking Google Fonts..."
+unzip -q fonts.zip
+cd fonts-main
+
+echo "Installing Google Fonts..."
+mkdir -p "$FONTS_DIR"
+find . -type f \( -iname "*.ttf" -o -iname "*.otf" \) -exec cp {} "$FONTS_DIR" \;
```

Comment:
Better (just as many `cp` as absolutlely needed, not as many as there are fonts to copy):
```bash
find . -type f \( -iname "*.ttf" -o -iname "*.otf" \) -print0 | xargs -0 cp -t "$FONTS_DIR"
```

Reply:
And if source and destination are inthe same file system, `mv` makes this lighting fast, as it is a simple rename, no copy involved (if not that `mv` decays to a `cp` followed by an `rm`, not much to loose here)
```bash
find . -type f \( -iname "*.ttf" -o -iname "*.otf" \) -print0 | xargs -0 -r mv-t "$FONTS_DIR"
```